server-common/src/main/java/org/apache/gravitino/server/http/RequestContext.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.server.http;
+
+/**
+ * RequestContext is used to store the context information of an HTTP request, including the userId.
+ */
+public interface RequestContext {
+
+  String getUserId();
+
+  void setUserId(String userId);
```

Comment:
@jerqi please help review this, thanks.

Reply:
Maybe we should extract the user information from the header. You can see `AuthenticationFilter`.

Reply:
That's also possible, but in that case, we need to add the user information header to all REST interfaces.

Reply:
> That's also possible, but in that case, we need to add the user information header to all REST interfaces.

I have added the information to every interface.